main_server/server/c_comdef_server.class.php

```diff
@@ -3038,7 +3038,12 @@ public static function GetLocalStrings( $in_lang_enum = null  ///< An enumeratio
                 if (isset($default_timezone) && $default_timezone) {
                     date_default_timezone_set($default_timezone);
                 }
-            
+                // Due to NAWS database limitations, only 5 formats at most can be exported. auto-config.inc.php can specify an optional variable $naws_export_formats_at_front
+                // that is an array of formats that should be exported first, to make sure they are included. This is particularly important for VM (Virtual Meeting),
+                // TC (Temporarily Closed) and HYBR (Hybrid), since these are essential and we don't want them falling off the end. If this variable isn't set in
+                // auto-config.inc.php, give it a default value that includes those plus a few others. If the meeting has any of these formats, they are exported
+                // first, in the order they appear in naws_export_formats_at_front.
+                c_comdef_server::$server_local_strings['naws_export_formats_at_front'] = isset($naws_export_formats_at_front) ? $naws_export_formats_at_front : ['VM', 'TC', 'HYBR', 'W', 'M', 'GL'];
```

Comment:
Is this something that we would ever want someone to override?

Also, what happens if the format code is different for another language here?

Reply:
I believe these are the NAWS mappings, not the translated BMLT formats, so translation is not an issue.

I don't think overridability is important here.

Reply:
But... if the PR has already built in the ability to override, I think it's fine to leave it.

Reply:
Right, these are NAWS mappings.  Regarding the ability to override, it does seem very unlikely anyone would want to change it. The first 3 (VM, TC, and HYBD) seem obvious and noncontroversial. Since it's programmed to allow additional preferences, I included Women, Men, and LGBT (with lower priority for inclusion, but since you get 5 these will always be included as well).  For these I can imagine someone getting bent out of shape -- we had some long running discussion in my area about the BMLT format codes for LGBT meetings for example -- so I made it overridable so that someone could change it if they care about this.  For example it seems more useful to know if it's a Women's meeting than if it's a candlelight meeting.  Could leave off LGBT ... dunno.